### DISCUSSION TITLE
error when starting Hyprland

### DISCUSSION THREAD

**Discussion by Dolbaeb777K** [end-4/dots-hyprland#287] [Extra Distros]
It's been a few days now, and I'm still trying to install dots-hyprland on opensuse tumbleweed. I installed AGS, but when entering ./update-ags.sh an error occurs. I tried to install cliphist, but there was simply no such package for opensuse tumbleweed. And I don't know what to do. After all. I'm quite new to Linux and I could put it on Arch, but the more I try to install it on a different distro, the more I get to know linux) So, I will be noble if you help!

```
[LOG] Created a bezier curve, baked 255 points, mem usage: 4.08kB, time to bake: 43.11µs. Estimated average calc time: 0.01µs.
[LOG] Registered dynamic var "$SLURP_COMMAND" -> "$(slurp -d -c f8daeeBB -b 55405044 -s 00000000)"
Authorization required, but no authorization protocol specified

amdgpu: amdgpu_cs_ctx_create2 failed. (-13)
/bin/sh: строка 1: fcitx5: команда не найдена
** Message: 07:41:53.254: couldn't access control socket: /run/user/1000/keyring/control: Нет такого файла или каталога
discover_other_daemon: 0/bin/sh: строка 1: /usr/lib/polkit-gnome/polkit-gnome-authentication-agent-1: Нет такого файла или каталога
/bin/sh: строка 1: hypridle: команда не найдена
ok
Failed to spawn cliphist: No such file or directoryFailed to spawn cliphist: No such file or directory
(com.github.Aylur.ags:3029): Gjs-CRITICAL **: 07:41:53.332: JS ERROR: Error: Requiring GtkLayerShell, version none: Typelib file for namespace 'GtkLayerShell' (any version) not found
require@resource:///org/gnome/gjs/modules/esm/gi.js:16:28
@gi://GtkLayerShell/:3:25


(com.github.Aylur.ags:3029): Gjs-CRITICAL **: 07:41:53.332: Module file:///usr/local/bin/ags threw an exception
Error: "Failed to connect to socket: Connection refused (os error 111)"
WARNING: socket file /run/user/1000/swww.socket was not deleted when the previous daemon exited
WARNING: failed to load cache for output X11-1: failed to open image: No such file or directory (os error 2)

```
![msg-1001725405168-143](https://github.com/end-4/dots-hyprland/assets/126266582/73009f51-5887-4bbf-9be0-cc7847297a43)

**Comment by clsty** [comment]
According to your log, there're at least 3 dependencies not installed properly: `gtk-layer-shell`, `cliphist`, `swww`.

This is all I know as I'm using Arch, not openSUSE.

> After all. I'm quite new to Linux and I could put it on Arch, but the more I try to install it on a different distro, the more I get to know linux

If you want get to know more about Linux, I recommend you to use Arch Linux and refer to [Arch Wiki](https://wiki.archlinux.org) frequently, then you will learn a lot. Trying different distros hardly helps.
